README.md

```diff
@@ -136,14 +136,22 @@ See [BREAKING](./BREAKING.md).
 If you want to contribute to Authelia, check the documentation available
 [here](https://docs.authelia.com/contributing/).
 
-## Contributors
-
-Authelia exists thanks to all the people who contribute. [[Contribute](./CONTRIBUTING.md)].
+Remember that Authelia exists thanks to all the people who contribute so don't be shy,
+come chat with us on [Matrix](#matrix) and start contributing too. [[Contribute](./CONTRIBUTING.md)].
 <a href="https://github.com/authelia/authelia/graphs/contributors"><img src="https://opencollective.com/authelia-sponsors/contributors.svg?width=890" /></a>
 
+## Why Open Source?
+
+You might wonder why Authelia is open source while it adds a great deal of security and user experience to your infrastructure at zero cost. It is open source because we deeply think that security should be brought to the crowd in order for everyone to be prepared to face the battlefield of the Internet
```

Comment:
```suggestion
You might wonder why Authelia is open source while it adds a great deal of security and user experience to your infrastructure at zero cost. It is open source because we firmly believe that security should be available for all to benefit in the face the battlefield which is the Internet
```